bleak/backends/bluezdbus/client.py

```diff
@@ -469,9 +476,14 @@ def mtu_size(self) -> int:
 
     # GATT services methods
 
-    async def get_services(self, **kwargs) -> BleakGATTServiceCollection:
+    async def get_services(
+        self, use_cached: bool = False, **kwargs
+    ) -> BleakGATTServiceCollection:
         """Get all services registered for this GATT server.
 
+        Args:
+            use_cached (bool): Use cached services if available.
```

Comment:
This should probably be called something like `dangerous_use_bleak_cache`. Dangerous since is subject to the issues described in #882 and `bleak` because the caching is done by bleak (we use `use_cached` elsewhere to mean us something cached by the OS bluetooth stack).